app/views/curation_concerns/base/_relationships_collection_row.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr>
+<th><%= t(".#{model_name}", type: model_name) %></th>
+<td>
+  <% if collection.blank? %>
+    <p><%= t('.empty', type: model_name) %></p>
+  <% else %>
+    <ul class="tabular">
+      <% collection.each do |col| %>
+        <li class='attribute title'>
+          <%= link_to col.title.first, contextual_path(col, nil) %>
```

Comment:
What do you mean by not working properly. What are you seeing? It certainly should work.